### PR TITLE
Adding Layer number to bad module list

### PR DESCRIPTION
**Pull request by fioriNTU**
corrspondent to #14970 in 81X, this PR adds the layer number to a log file with a list of "bad modules" id. Used by Tracker Offline shifters

### CODE REVIEW

**Comment by dmitrijus** [comment]
please test

**Comment by dmitrijus** [comment]
+1
(if tests pass)

File: DQM/SiStripMonitorClient/src/SiStripTrackerMapCreator.cc [hunk 1]
```diff
@@ -363,8 +363,10 @@ void SiStripTrackerMapCreator::paintTkMapFromAlarm(uint32_t det_id, const Tracke
   if(isBad) { rval=255; gval=255; bval = 0; comment << " PCLBadModule ";}
   trackerMap_->setText(det_id, comment.str());
   trackerMap_->fillc(det_id, rval, gval, bval);
-  if(badmodmap && (flag!=0 || isBad)) (*badmodmap)[det_id] = comment.str();
-
+  uint  lay= tTopo->layer(SiStripDetId(det_id));
```
Comment (davidlange6) [inline DQM/SiStripMonitorClient/src/SiStripTrackerMapCreator.cc]: @fioriNTU - please put all three lines into the if statement. 'lay' and 'layer' are needed only in the if

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**